ui/components/app/assets/asset-list/asset-list-control-bar/asset-list-control-bar.tsx

```diff
@@ -303,13 +310,38 @@ const AssetListControlBar = ({
     }
 
     return t('popularNetworks');
-  }, [isTokenNetworkFilterEqualCurrentNetwork, currentMultichainNetwork, t]);
+  }, [
+    enabledNetworksByNamespace,
+    isTokenNetworkFilterEqualCurrentNetwork,
+    currentMultichainNetwork.isEvmNetwork,
+    currentMultichainNetwork.network.nickname,
+    currentMultichainNetwork?.nickname,
+    t,
+    allNetworks,
+  ]);
+
+  const singleNetworkIconUrl = useMemo(() => {
+    if (!isGlobalNetworkSelectorRemoved) {
+      return undefined;
+    }
+
+    const chainIds = Object.keys(enabledNetworksByNamespace);
+
+    if (chainIds.length !== 1) {
+      return undefined;
+    }
+
+    return currentMultichainNetwork?.network?.rpcPrefs?.imageUrl;
+  }, [
+    currentMultichainNetwork?.network?.rpcPrefs?.imageUrl,
+    enabledNetworksByNamespace,
+  ]);
 
   return (
     <Box
       className="asset-list-control-bar"
-      marginLeft={2}
-      marginRight={2}
+      marginLeft={4}
+      marginRight={4}
```

Comment:
Modified padding so the network dropdown is inline with other content.

<img width="267" height="434" alt="Screenshot 2025-08-20 at 22 01 44" src="https://github.com/user-attachments/assets/abaedde7-990f-4ff8-9625-32b7cd4618f3" />